contracts/Staking.sol

```diff
@@ -447,6 +462,19 @@ contract Staking is Autopetrified, ERCStaking, ERCStakingHistory, IStakingLockin
         emit Staked(_accountAddress, _amount, newStake, _data);
     }
 
+    function _unstakeUnsafe(address _from, uint256 _amount, bytes _data) internal {
```

Comment:
I think it would be more intuitive `_unsafeUnstake`, both sounds weird tho haha

Reply:
We already have some other functions in this contract with the `Unsafe` suffix. I put as a suffix because it’s less “distracting”. Otherwise we would have 3 functions starting with the same `_unsafe` prefix, which IMO makes it harder to read. But if you really like it better the other way around let me know and I’ll change the three functions.